src/packages/console/src/pages/homePage.tsx

```diff
@@ -122,53 +85,50 @@ export const HomePage = (): React.ReactElement => {
             accounts.map((account: Account, index: number): React.ReactElement => (
-              <Paper key={index} elevation={0} className={classes.accountBox}>
-                <Box width={1} display='flex' justifyContent='start' alignItems='baseline'>
-                  <Typography variant='h5' className={classes.accountName}>
+              <Box variant='bordered' isFullHeight={true} key={index}>
+                <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center}>
+                  <Text variant='header5'>
                     {account.name}
-                  </Typography>
-                  <Typography color='textSecondary' className={classes.accountType}>{`(${account.accountType})`}</Typography>
-                  {authManager.getHasJwtPermission(`acc-${account.accountId}-ed`) && <Button color='primary' onClick={(): void => onManageAccountClicked(account)}>Manage</Button>}
-                  <Box flexGrow={1} />
-                  {authManager.getHasJwtPermission(`acc-${account.accountId}-adm`) && <Button color='primary' onClick={(): void => onCreateSiteClicked(account)}>Create site</Button>}
-                </Box>
-                <Grid container spacing={2} className={classes.siteCardGrid}>
-                  {accountSites[account.accountId].map((site: Site, innerIndex: number): React.ReactElement => (
-                    <Grid item xs={12} sm={6} md={4} lg={3} key={innerIndex}>
-                      <SiteCard site={site} onSiteClicked={onSiteClicked} isEnabled={authManager.getHasJwtPermission(`st-${site.siteId}-vw`)} />
-                    </Grid>
+                  </Text>
+                  <Text variant='colored'>{`(${account.accountType})`}</Text>
+                  {authManager.getHasJwtPermission(`acc-${account.accountId}-ed`) && <Button onClicked={(): void => onManageAccountClicked(account)} text='Manage' />}
+                  <Stack.Item growthFactor={1} shrinkFactor={1} />
+                  {authManager.getHasJwtPermission(`acc-${account.accountId}-adm`) && <Button onClicked={(): void => onCreateSiteClicked(account)} text='Create site' />}
+                </Stack>
+                <Spacing variant={PaddingSize.Wide} />
+                <EqualGrid childSizeResponsive={{ small: 4, base: 3 }} contentAlignment={Alignment.Start} shouldAddGutters={true}>
+                  {accountSites[account.accountId].map((site: Site, innerIndex: number) => (
+                    <SiteCard key={innerIndex} site={site} onSiteClicked={onSiteClicked} isEnabled={authManager.getHasJwtPermission(`st-${site.siteId}-vw`)} />
                   ))}
                   {accountSites[account.accountId].length === 0 && (
-                    <Grid item xs={12}>
-                      <Typography color='textSecondary'>
-                        {'No sites yet. Create one now!'}
-                      </Typography>
-                    </Grid>
+                    <Text variant='header6-note'>
```

Comment:
should not be a header

Reply:
So what variant should I use to display a grey color text with normal font size

Reply:
`light`, see the other psges i updated

Reply:
Done